site/.markdownlint.jsonc

```diff
@@ -0,0 +1,8 @@
+{
+  "line-length": false,
+  "no-duplicate-heading": false,
```

Comment:
Where does this fail?

Reply:
![Screenshot from 2023-07-07 16-16-52](https://github.com/grammyjs/website/assets/75927667/f3bf44bb-c5dc-4895-8a44-6bc3d0957f0b)

Reply:
I was referring to `no-duplicate-heading`

Reply:
But why does `no-duplicate-heading` fail for long lines?

Reply:
Or were you talking about different things? I understand that in the Hosting/VPS section we have repeated headers, but that makes sense because we describe the same set of steps twice in two different contexts.

All the other reported cases can be fixed, noß

Reply:
No, this rule does not complain about long headlines. It's just that you gave an example of code for two rules, and I responded accordingly.

Reply:
Ah well yes, GH always renders a few contextual lines of code before, but I added my comment to line 3 and I meant to refer to `no-duplicate-heading` only :)

Reply:
In `hosting /vps.md` we can insert a comment at the beginning of the page to disable the `MD024/no-duplicate-heading/no-duplicate-header` rule globally on the page. On other pages, you should rephrase the headers:

- `plugins/keyboard.md`. Complaint about the repeated header `Responding to Clicks`, which occurs in the context of the inline and custom keyboard. At the same time, headers such as `Building ...`, `Sending ...` and `Removing ...` are used with the keyboard in mind: `Building an Inline Keyboard' and `Building a Custom Keyboard', for example. You can do the same with a problematic header: `Responding to Clicks on the Inline Keyboard` and `Responding to Clicks on the Custom Keyboard`.
- `hosting/heroku.md`. Complaint about the repeated header `Creating bot.ts`, which occurs in the context of webhooks and a long polling. We can do the same thing here as I suggest for `plugins/keyboard.md`, i.e. create the headers `Creating bot.ts for Webhooks` and `Creating bot.ts for a Long Polling`.
- There are still a few pages (`id/plugins/conversations.md`, `es/guide/filter-queries.md`) where translators have somehow duplicated certain headings, although they are all unique in the original.

Reply:
I'll fix these things in a subsequent PQ targeting this branch.

Reply:
Okay, I'll do it.

Reply:
Was this issue resolved?

Reply:
Work in progress https://github.com/grammyjs/website/pull/908